src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/RecoveryServicesBackupCmdletBase.cs

```diff
@@ -121,9 +121,9 @@ private void HandleException(Exception exception)
             }
         }
 
-        public override void ExecuteCmdlet()
+        protected override void BeginProcessing()
         {
```

Comment:
No need to override in every cmdlet. This would be run once per cmdlet by the ps infra itself.